test/import/util.ts

```diff
@@ -24,6 +24,10 @@ export async function expectImportMatchSnapshot(fn: () => ImportBase, options?:
     for (const lang of languages) {
       const jsiiPath = lang === Language.TYPESCRIPT ? path.join(workdir, '.jsii') : undefined;
 
+      if (lang === Language.GO) {
+        fs.writeFileSync(path.join(workdir, 'go.mod'), 'module integtest');
+      }
```

Comment:
I added this in case we do want to add go snapshot tests at some point. Right now though, snapshot takes very long and if I add Go to the language list on line 22, then it takes 8+ minutes to run a full build+test on my machine, so I wasn't sure if it's worth the cost. Just say the word and I can add them though 🙂

Reply:
I'm ok with skipping snapshots for go for now - feels like we should add all languages at some point and maybe parallelize in the workflow somehow.